slu-scripts/src/index.spec.ts

```diff
@@ -6,6 +6,7 @@ import yargs from 'yargs';
 import * as vpuf from './vpuf/vpuf';
```

Comment:
fs module import seems to be unused

Reply:
it is used in the afterAll() for deleting the created files during the tests.